core-feature-pack/src/main/resources/content/bin/common.ps1

```diff
@@ -0,0 +1,256 @@
+if($PSVersionTable.PSVersion.Major -lt 2) {
+    Write-Warning "posh-git requires PowerShell 2.0 or better; you have version $($Host.Version)."
```

Comment:
Ups, that was copy paste from some local script is use for git command line completion.
Forgot to update the message.